lib/presentation/engage/bible_series/widgets/scripture_body.dart

```diff
@@ -10,6 +12,81 @@ class ScriptureContentBodyWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Container();
+    return Container(
+        width:
+            1 * MediaQuery.of(context).size.width, //change depending on parent
+        padding: const EdgeInsets.fromLTRB(0, 16, 0, 16),
+        margin: const EdgeInsets.all(20),
+        child: Column(
+          children: [
+            ListView.builder(
+                shrinkWrap: true,
+                physics: ClampingScrollPhysics(),
+                padding: const EdgeInsets.only(top: 0),
+                itemCount: scriptureContentBody.properties.scriptures.length,
+                itemBuilder: (BuildContext context, int index) {
+                  //return Text("top");
+                  return scripture(
+                      scriptureContentBody.properties.scriptures[index]);
+                }),
+            Align(
+                alignment: Alignment.centerLeft,
+                child: copyright(scriptureContentBody))
+          ],
+        ));
   }
 }
+
+Widget scripture(Scripture script) {
+  return Column(
+    children: [
+      Padding(
+        padding: const EdgeInsets.fromLTRB(0, 8, 0, 16),
+        child: title(script),
+      ),
+      ListView.builder(
+        padding: const EdgeInsets.only(top: 0),
+        shrinkWrap: true,
+        itemCount: script.verses.length,
+        physics: ClampingScrollPhysics(),
+        itemBuilder: (BuildContext context, int index) {
+          String key = script.verses.keys.elementAt(index);
+          return Padding(
+            padding: const EdgeInsets.only(bottom: 8.0),
+            child: verse(script, key),
+          );
+        },
+      ),
+    ],
+  );
+}
+
+Widget title(Scripture script) {
```

Comment:
There is also an optional `title` parameter in the Scripture class that should be used here. Sorry if it's not included in the Figma. You can put it just below the verse information.

Reply:
I updated the Figma to reflect this.

Reply:
ok got it